packages/solid/components/Input/Input.types.ts

```diff
@@ -37,6 +37,16 @@ export interface InputProps extends UIComponentProps, InputStyleProperties {
    */
   titleSignal: Signal<string>;
 
+  /**
+   * general styles applied to input `View` container
+   */
+  style?: unknown;
+
+  /**
+   * specific styles applied to input `Text` item
+   */
+  textStyle?: TextStyles;
```

Comment:
can we update `@lightningtv/solid` import to include `TextStyles` as well? This isn't currently imported